tmva/sofie/test/CMakeLists.txt

```diff
@@ -148,34 +127,8 @@ if (ROOT_TORCH_FOUND)
   endif()
 endif()
 
-add_executable(emitGNN
-              GNN/EmitGNN.cxx
-              ${CMAKE_SOURCE_DIR}/tmva/sofie/src/RModel_GNN.cxx
-              ${CMAKE_SOURCE_DIR}/tmva/sofie/src/SOFIE_common.cxx
-)
-target_include_directories(emitGNN PRIVATE
-   ${CMAKE_SOURCE_DIR}/tmva/sofie/inc
-   ${CMAKE_SOURCE_DIR}/tmva/inc
-   ${CMAKE_SOURCE_DIR}/core/foundation/inc
-   ${CMAKE_BINARY_DIR}/ginclude   # this is for RConfigure.h
-)
-target_link_libraries(emitGNN ROOTTMVASofie)
-set_target_properties(emitGNN PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
-## silence protobuf warnings seen in version 3.0 and 3.6. Not needed from protobuf version 3.17
-target_compile_options(emitGNN PRIVATE -Wno-unused-parameter -Wno-array-bounds)
-
-add_executable(emitGraphIndependent
-              GNN/EmitGraphIndependent.cxx
-              ${CMAKE_SOURCE_DIR}/tmva/sofie/src/RModel_GraphIndependent.cxx
-              ${CMAKE_SOURCE_DIR}/tmva/sofie/src/SOFIE_common.cxx
-)
-target_include_directories(emitGraphIndependent PRIVATE
-   ${CMAKE_SOURCE_DIR}/tmva/sofie/inc
-   ${CMAKE_SOURCE_DIR}/tmva/inc
-   ${CMAKE_SOURCE_DIR}/core/foundation/inc
-   ${CMAKE_BINARY_DIR}/ginclude   # this is for RConfigure.h
-)
-target_link_libraries(emitGraphIndependent ROOTTMVASofie)
-set_target_properties(emitGraphIndependent PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
-## silence protobuf warnings seen in version 3.0 and 3.6. Not needed from protobuf version 3.17
-target_compile_options(emitGraphIndependent PRIVATE -Wno-unused-parameter -Wno-array-bounds)
+ROOT_EXECUTABLE(emitGNN GNN/EmitGNN.cxx LIBRARIES ROOTTMVASofie)
+ROOT_ADD_TEST(tmva-sofie-EmitGNN COMMAND emitGNN)
+
+ROOT_EXECUTABLE(EmitGraphIndependent GNN/EmitGraphIndependent.cxx LIBRARIES ROOTTMVASofie)
+ROOT_ADD_TEST(tmva-sofie-EmitGraphIndependent COMMAND EmitGraphIndependent)
```

Comment:
Is it safe to not use the ROOTTEST_GENERATE_EXECUTABLE macro in this case and thus not use the FIXTURES_* ?